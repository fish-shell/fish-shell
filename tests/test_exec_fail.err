<W> fish: An error occurred while redirecting file 'nosuchfile'
open: No such file or directory
<W> fish: An error occurred while redirecting file 'nosuchfile'
open: No such file or directory
