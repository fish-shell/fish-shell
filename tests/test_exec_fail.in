exec cat < nosuchfile
echo "failed: $status"
not exec cat < nosuchfile
echo "neg failed: $status"
exec cat < /dev/null
echo "not reached"
