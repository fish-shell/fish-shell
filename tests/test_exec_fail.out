failed: 1
neg failed: 0
